Library/BarPlotsCombineTwoAxes.Rmd

```diff
@@ -29,6 +31,17 @@ data <- read_dta("https://github.com/worldbank/r-econ-visual-library/raw/master/
 # Printing the first five rows of the data to visualize its structure before data wrangling
 head(data)
 
+# Data Dictionary ---------------
+  ## Data Source: Christian, Paul, and Brian Dillon. 2018. “Growing and Learning When Consumption Is Seasonal: Long-Term Evidence From Tanzania.” Demography 55 (3): 1091–1118. doi:10.1007/s13524-018-0669-4.
+  ## Tidy format? Yes
+  ## Unit of Observation: Group-level mean, collapsed by month of interview and food type
+  ## Variables of interest:
+    # int1mo: Month of interview identifier 
+    # food_group: Food group identifier 
+    # number_group: Mean number of food type consumed in given month 
+    # total_exp: Total expenditures on food group in given month
+
+
 # Data wrangling ---------------
```

Comment:
The data wrangling in this graph is not particularly useful. Let's save a new dataset that already incorporates this wrangling and start from it.